tests/utils/test_file_lookups.py

```diff
@@ -50,58 +49,51 @@ def test_project_id_exception(mock_gitlab):
 @mock.patch('foremast.utils.lookups.gitlab')
 def test_project_id_success(mock_gitlab):
     """Check resolving GitLab Project ID is successful."""
-    project_id = 30
-
-    mock_gitlab.Gitlab.return_value.getproject.return_value = {'id': project_id}
-
+    project_id = object
+    mock_gitlab.Gitlab.return_value.projects.get.return_value = object
     seeker = FileLookup()
-
-    assert seeker.project_id == project_id
+    assert seeker.project == project_id
 
 
 @mock.patch('foremast.utils.lookups.gitlab')
 def test_get(gitlab):
     """Check _get_ method."""
     my_git = FileLookup()
 
-    my_git.server.getfile.return_value = {'content': base64.b64encode(TEST_JSON_BYTES)}
-
-    result = my_git.get()
-    assert isinstance(result, str)
-    assert TEST_JSON == my_git.get()
+    with mock.patch.object(FileLookup, 'remote_file', return_value=TEST_JSON) as mock_method:
+        result = my_git.get()
+        assert isinstance(result, str)
+        assert TEST_JSON == my_git.get()
 
 
 @mock.patch('foremast.utils.lookups.gitlab')
 def test_json(gitlab):
     """Check _json_ method."""
     my_git = FileLookup()
 
-    my_git.server.getfile.return_value = {'content': base64.b64encode(TEST_JSON_BYTES)}
-
-    result = my_git.json()
-    assert isinstance(result, dict)
-    assert result['ship'] == 'pirate'
+    with mock.patch.object(FileLookup, 'get', return_value=TEST_JSON) as mock_method:
```

Comment:
Use decorator and `assert_called_with()` like above.

Reply:
Fixed. Definitely makes the tests easier to read.